test/powershell/Language/Operators/ComparisonOperator.Tests.ps1

```diff
@@ -47,9 +47,13 @@ Describe "ComparisonOperator" -tag "CI" {
 	 !0       | Should Be $true
     }
 
-	It "Should be $true for 'Hello','world' -contains 'Hello'" {
-	 $arr= 'Hello','world'
-	 $arr -contains 'Hello'       | Should Be $true
+	It "Should be <result> for 'Hello','world' <operator> <rhs>" -TestCases @(
+        @{result = $true; operator = "-contains"; rhs = "'Hello'"},
```

Comment:
Test for case sensitive variants? ccontains and cnotcontains? There are some tests below. Can we move them to this It and add it as part of -TestCases array.